core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -2654,7 +2655,7 @@ class ReplicaManager(val config: KafkaConfig,
     config.replicaSelectorClassName.map { className =>
       val tmpReplicaSelector: ReplicaSelector = Utils.newInstance(className, classOf[ReplicaSelector])
       tmpReplicaSelector.configure(config.originals())
-      Plugin.wrapInstance(tmpReplicaSelector, metrics, className)
+      Plugin.wrapInstance(tmpReplicaSelector, metrics, ReplicationConfigs.REPLICA_SELECTOR_CLASS_CONFIG)
```

Comment:
This fixes an issue we introduced in https://github.com/apache/kafka/commit/79fe1305b62498cc6e9a15b2163134f0df3e1757